crates/anvil/src/eth/backend/mem/mod.rs

```diff
@@ -2004,6 +2009,46 @@ impl Backend {
         Ok(None)
     }
 
+    // Returns the traces matching a given filter
+    pub async fn trace_filter(
+        &self,
+        filter: TraceFilter,
+    ) -> Result<Vec<LocalizedTransactionTrace>, BlockchainError> {
+        let matcher = filter.matcher();
+        let start = filter.from_block.unwrap_or(0);
+        let end = filter.to_block.unwrap_or(self.best_number());
```

Comment:
Could be good to include a sanity check here to make sure end > start

https://github.com/paradigmxyz/reth/blob/89d0281ce3dd06e57c18b226df84af8809b750cd/crates/rpc/rpc/src/trace.rs#L262-L266

Reply:
Good catch. Updated with the test.